### PR TITLE
tweaks

### PR DESCRIPTION
**Pull request by SpacePossum**
- Normalize naming of the fixer `PHP CS Fixer` -> `PHP-CS-Fixer` following the convension used in the README 
- some other minors

Please consider picking this commit on your branch :)

### CODE REVIEW

File: COOKBOOK-FIXERS.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
-Cookbook - Making a new Fixer for PHP CS Fixer
+Cookbook - Making a new Fixer for PHP-CS-Fixer
```
Comment (keradus) [inline COOKBOOK-FIXERS.md:1]: :-1:

http://fabien.potencier.org/php-cs-fixer-finally-reaches-version-1-0.html
name should not contain `-`, only file should

Comment (SpacePossum) [reply COOKBOOK-FIXERS.md:1]: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.12/README.rst#globally-composer

```
To install PHP-CS-Fixer,
```

Comment (SpacePossum) [reply COOKBOOK-FIXERS.md:1]: btw. I don't care which version we use, but would be nice to be consistent ;)

Comment (keradus) [reply COOKBOOK-FIXERS.md:1]: sure, I would just go with normal naming, PHP CS Fixer.

**Comment by keradus** [comment]
merge commit didn't go well.. please clean up this PR ;)

**Comment by keradus** [comment]
@SpacePossum ?

**Comment by SpacePossum** [comment]
I'll clean this up when Im off from work, I hope

**Comment by keradus** [comment]
then maybe just pick @g105b and yours commit and recreate PR, it may be faster

**Pull request closed without merging**